autosklearn/estimators.py

```diff
@@ -795,14 +797,18 @@ def fit(self, X, y,
         """
         # Before running anything else, first check that the
         # type of data is compatible with auto-sklearn. Legal target
-        # types are: continuous, binary, multiclass.
+        # types are: continuous, continuous-multioutput, 'multiclass'
+        # binary
         target_type = type_of_target(y)
-        if target_type in ['multiclass-multioutput',
-                           'multilabel-indicator',
-                           'unknown',
-                           ]:
-            raise ValueError("regression with data of type %s is not"
-                             " supported" % target_type)
+        supported_types = ['continuous', 'binary', 'multiclass', 'continuous-multioutput']
+        if target_type not in supported_types:
+            raise ValueError("Regression with data of type {} is "
```

Comment:
Same as above, please update the error message to include a link to scikit-learn.

Reply:
Understood, I will add it in the next push.